config/main.py

```diff
@@ -3916,6 +3916,172 @@ def del_user(db, user):
             click.echo("Restart service snmp failed with error {}".format(e))
             raise click.Abort()
 
+#
+# 'bmp' group ('config bmp ...')
+#
+
+@config.group()
+@click.pass_context
+def bmp(ctx):
+    """BMP-related configuration"""
+
+#
+# 'enable' subgroup ('config bmp enable ...')
+#
+@bmp.group()
+@click.pass_context
+def enable(ctx):
+    """Enable BMP table dump """
+
+#
+# 'bgp-neighbor-table' command ('config bmp enable bgp-neighbor-table')
+#
+@enable.command('bgp-neighbor-table')
+@click.pass_context
+def enable_bgp_neighbor_table(ctx):
+    """enable bgp-neighbor-table sessions
+       In the case of Multi-Asic platform, corresponding database instance of neighbor will be operated.
+    """
+    log.log_info("'bmp enable bgp-neighbor-table' executing...")
+    config_db = ConfigDBConnector()
+    config_db.connect()
+    try:
+        bmp_table = config_db.get_table('BMP')
```

Comment:
>BMP [](http://example.com/codeflow?start=41&length=3)

This table is not inside your HLD. You mentioned `FEATURE|bmp` table there.

Reply:
or you need to iterate the HLD PR.

Reply:
feature switch config will be in https://github.com/sonic-net/sonic-buildimage/pull/18946

Reply:
My point is that this is ConfigDB design, and your HLD does not mention the design.

Reply:
HLD contains this config cli design, refer below
<img width="1075" alt="image" src="https://github.com/sonic-net/sonic-utilities/assets/97083744/23dcd5c6-7f4e-4ac9-bbfd-2666d851838f">

Reply:
Thanks! above screenshot is the `config` CLI design. My point is that the HLD should has a ConfigDB schema design. Since you already has [https://github.com/sonic-net/sonic-buildimage/pull/18530](https://github.com/sonic-net/sonic-buildimage/pull/18530) merged. I will close this commemnt.